doc/examples/reconst_mapmri.py

```diff
@@ -12,24 +12,30 @@
 
 from dipy.reconst.mapmri import MapmriModel
 from dipy.viz import fvtk
-from dipy.data import fetch_sherbrooke_3shell, read_sherbrooke_3shell, get_sphere
+from dipy.data import fetch_cenir_multib, read_cenir_multib, get_sphere
 from dipy.core.gradients import gradient_table
+import matplotlib.pyplot as plt
 
 """
 Download and read the data for this tutorial.
 
-fetch_isbi2013_2shell() provides data from the ISBI HARDI contest 2013 acquired 
-for two shells at b-values 1500 and 2500.
+MAPMRI requires multi-shell data, to properly fit the radial part of the basis.
+The total size of the downloaded data is 1760 MBytes, however you only need to
+fetch it once. Parameter ``with_raw`` of function ``fetch_cenir_multib`` is set
+to ``False`` to only download eddy-current/motion corrected data:.
+"""
+
+fetch_cenir_multib(with_raw=False)
 
-The six parameters of these two functions define the ROI where to reconstruct
-the data. They respectively correspond to (xmin,xmax,ymin,ymax,zmin,zmax)
-with x, y, z and the three axis defining the spatial positions of the voxels.
 """
+For this example we select only the shell with b-values equal to the one of the
+Human Connectome Project (HCP).
+"""
+bvals = [1000, 2000, 3000]
 
-fetch_sherbrooke_3shell()
-img, gtab = read_sherbrooke_3shell()
+gtab, img = read_cenir_multib(bvals)
```

Comment:
Please rebase on top of master. I believe this should now be:
    
    img, gtab = read_cenir_multib(bvals)